New Race: Canine Race--Vulpkanin

Fox-McCloud opened the pull request:
You've heard about it for a while, and now, it's finally here.

![vulpkanin1](https://www.dropbox.com/s/zq21gsencnqper2/Ca23111.png?raw=1)

First off, a BIG, gigantic, huge thanks to Full of Skittles for the sprites for these guys; this project literally would have never happened without her; be sure to give her a big thanks, shoutout, or hug (in-game) if you see her; she deserves 100% of the credit for the visual design.

- Adds in a canine race, known as the "Vulpkanin". These are meant to be a *generic* canine race--not specifically a wolf, fox, husky, or what have you--meaning if you want yours to be a husky, that's great--if you want yours to be a fox...that's great too; the nature of species is they're diverse; roleplay flexibility for the win!

Little background blurb about them: 

- Vulpkanin are a species of extremely sharp-witted canine-pideds residing on the planet Altam just barely within the dual-star Vazzend system. Their politically de-centralized society and independent natures have led them to become a species and culture both feared and respected for their scientific breakthroughs. Discovery, loyalty, and utilitarianism dominates their lifestyles to the degree it can cause conflict with more rigorous and strict authorities. They speak a guttural language known as 'Canilunzt' which has a heavy emphasis on utilizing tail positioning and ear twitches to communicate intent.

Special Traits/Abilities

- Tail wagging! (the most important thing)

Other notes

 - For the most part, they're feature complete; you can color their fur, eyes, hair. They have 15 default hairstyles and 8 facial hair styles to select from (that's more than Tajarans have!)---they have their own language...generally features of what you'd expect of a race.. Additionally, they have a lesser form, so  when it comes time to be cloned or have a monkified version of a Vulpkanin, that's more than possible.

Future

- More potential hair and facial hairstyles to come.
- potential tail customization

Cost

- They'll be freely available to anyone and everyone.

Iamgoofball commented:
>tail wag
>faster movespeed
>extra brute and burn

so reskinned tajarans

why add them if it's reskinned tajarans

Iamgoofball commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -412,6 +412,18 @@
+	New()
+		..()
+		for(var/i = 1; i <= 5; i++)
+			new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/wolpincube(src)
```
standardize this, make the created type a var on a base /obj/item/weapon/storage/box/cube object, and make the various renamed monkey cube boxes a subtype of that

Fox-McCloud replied:
That can be done in the future, but...outside of the scope of this PR, especially given literally everything else in this file is also relatively pathed.

Iamgoofball commented on code/modules/client/preferences.dm:
```diff
@@ -352,7 +352,7 @@ datum/preferences
 				dat += "<a href='?_src_=prefs;preference=eyes;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_eyes, 2)][num2hex(g_eyes, 2)][num2hex(b_eyes, 2)]'><table  style='display:inline;' bgcolor='#[num2hex(r_eyes, 2)][num2hex(g_eyes, 2)][num2hex(b_eyes)]'><tr><td>__</td></tr></table></font><br>"
 
-				if(species == "Unathi" || species == "Tajaran" || species == "Skrell" || species == "Slime People")
+				if(species == "Unathi" || species == "Tajaran" || species == "Skrell" || species == "Slime People" || species == "Vulpkanin")
```
why not change this to a list of acceptable species strings

minor standardization nitpick: why is a backend var using a frontend name

Iamgoofball commented on code/modules/mob/language.dm:
```diff
@@ -180,6 +180,20 @@
+/datum/language/vulpkanin
+	name = "Canilunzt"
+	desc = "The guttural language spoken and utilized by the inhabitants of Vazzend system, composed of growls, barks, yaps, and heavy utilization of ears and tail movements.Vulpkanin speak this language with ease."
+	speech_verb = "rawrs"
```
>rawrs

really

Fox-McCloud replied:
I tried to keep it within the same vein as the Tajaran's speech modifiers, which are "mrowl" and "yowl".

Iamgoofball replied:
which were set to that because "muh cute x3 *POMF*", and is still unprofessional

Iamgoofball commented on code/modules/mob/living/carbon/species.dm:
```diff
@@ -410,6 +410,38 @@
+	darksight = 8
+	slowdown = -1
+	brute_mod = 1.1
+	burn_mod = 1.1
```
these need to be 1.5, or maybe even 2

1.1 just isn't enough for a race that when stacking meth, morphine, and ephedrine can get -4 movespeed

Fox-McCloud replied:
Claws don't do extra damage compared to punches; that's one thing that got standardized away a while back.

In any event, this is definitely something I'll be keeping a close eye on, especially considering some of the things you mentioned (which will make it harder to gather data on since hardly anyone here uses meth here, routinely).

But yeah, it's a concern of mine, as well.

Iamgoofball replied:
>hardly anyone uses meth

u wot m8

Iamgoofball commented on code/modules/reagents/reagent_containers/food/snacks.dm:
```diff
@@ -1753,6 +1753,13 @@
+	monkey_type = /mob/living/carbon/monkey/vulpkanin
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/wolpincube
+	name = "wolpin cube"
+	monkey_type = /mob/living/carbon/monkey/vulpkanin
```
honestly I'd do away with the wrapped feature entirely, it's pointless busywork really

Fox-McCloud replied:
Not within the scope of this PR.

Iamgoofball commented on code/modules/mob/living/carbon/species.dm:
```diff
@@ -410,6 +410,38 @@
+	brute_mod = 1.1
+	burn_mod = 1.1
+
+	blurb = "Vulpkanin are a species of extremely quick canine-pideds residing on the planet Altam just barely within the \
```
these blurbs should really be moved to serverside defines of some kind so that lore changes can be made by a server hoster easily and not require code changes

Fox-McCloud replied:
It's not really a major problem for us, nor is it within the scope of this PR =p

Zuhayr commented:
I feel like Bay is falling behind in the species variety race, we should really up our game. Ferret men when?

Iamgoofball commented:
@Zuhayr just port greys, plasmaman, and shadowpeople and you're set

fj45 commented:
![](https://cloud.githubusercontent.com/assets/6563111/5459026/fa64c184-8535-11e4-9cde-52278a43c1d7.png)

Iamgoofball commented:
@fj45 don't give them any ideas

Keilden commented:
Let the Tajaran Vulpkanin wars start!

PS kanin is swedish for rabbit.

Fox-McCloud commented:
@Keilden 

Interesting; I know "Tajar" is technically a Spanish word for cut/sharpen. Hard to get some word that vaguely relates to what you want without meaning something you don't want it to in another language.

Regen1 commented:
Catgirls when?

Keilden commented:
Surskit the pokemon is a way better name in sweden, Sur is swedish for sour and skit is shit.

Robertha89 commented:
it's more the fact that pierce, sharp and blunt each does damage in it's own way. The additional brute/burn mod seems like a nerf just to nerf em, while i can agree with the burn somewhat, the brute seems a bit silly.
Also, while it makes you slightly faster, if they are on meth there is pretty little you can do, and a good way to stack everything without od will be hard. There is also the fact that meth use is rare, as in as rare as the rare roundtypes rare.
I would say remove the mod's to begin with and see how it works ingame. Then reward/punish the playerbase by how they ab/use it or not.
Like they did with remove nymphs ability to pick up stuff.

Iamgoofball commented:
@robertha89 are you kidding me

if you remove the damage mods it's literally "human+"

there's no gameplay balance reason to play human if I could just play dogbeest and get bonus movespeed

Fox-McCloud commented:
Yeah, I'm not tacking on a movement speed modifier to a race without some downside to them--and as it stands, I'm highly skeptical of the movement speed buff of the race to begin with.

tigercat2000 commented:
so
a slightly-fast race
the same cost as slime people and vox
with none of the advantages they have



sounds fine to me

Robertha89 commented:
Well, it's more Tajaran+ since it still got all the tajaran weaknesses of heat, but resistant to cold, pluss it got the claws and everything. even at -3 you can't be caught, cause lolmeth, -4 makes literally no difference compared to -3 when you are on meth due to the 300% stun reduction alone.
I even don't see why it should have the movement speed bonus at all, it's basically just a re-skinned tajaran.

ZomgPonies commented:
Speed buffs are notoriously hard to balance for. I'm not sure how this'll play out.

PJB3005 commented:
:-1: 

Fucking dog catbeasts is what this is.

Mijhead commented:
These aren't canine, they're obviously vulpine, I believe the word you were looking for is canidae. Besides that, even their name is an obvious allusion to what they are; Humanoid foxes.

Finalsong commented:
^

IS THIS REAL LIIIIIIIIIIIIIIIIIIFE

IS THIS JUST HERESYYYYYYYYYYYYYYYYYYYY

TheDZD commented:
I honestly don't feel that 10% extra brute and burn damage is quite enough to make up for a -1 slowdown. Normally it would take 150 damage to hard-crit a person, and 200 to kill them. With the 10% increase, it would take approximately 136 to crit, and 181 to kill. Which is an just under a 1 laser shot difference each time.

Considering Greys have a 25% brute mod solely for gaining telepathy (darksight was broken by object-based lighting as far as I'm aware, so I am ignoring that), it would probably be better to give the race a 1.2 brute and burn mod rather than 1.1, considering the negative slowdown also makes it more difficult to actually catch them to hit with a melee weapon, and makes it a bit easier to dodge out of the way of projectiles.

Flip-Sandwich commented:
Can we get some images of the sprites that aren't mangled by stretch to fit?

Iamgoofball commented:
@Finalsong oh my god you didn't even quote the song right

Finalsong commented:
@Iamgoofball I forgot one word. Boohoo.

The heresy part is from http://nanotrasen.se/phpBB3/viewtopic.php?f=9&t=3311

Iamgoofball commented:
@Finalsong that's not even a creative parody

Finalsong commented:
@Iamgoofball Not MEANT to be creative.

Fox-McCloud commented:
@ZomgPonies  and @TheDZD 

Yep; it's a major concern of mine too; will discuss it further with you two and the coders.

As for them being vulpines; no they're not. They're meant to be generic canines--that's what I requested. The thing is, it's difficult to get a sprite that fits with existing clothing and designs without causing causing conflicts(ore requiring entirely customize clothing, like with Vox). Furthermore, given the diversity of dogs, it's difficult to also make a sprite that doesn't look like one of them.

Robertha89 commented:
When i was thinking about building dog people, i wanted to create a dectective like ability that could smell fingerprints and things like that with a sniff command, it got as far as i have no idea how to do this, so meh.

MactoPerFuror commented:
The species name I think might need a change. Vulpkanin is an ugly portmanteau, your practically saying both words fully almost. Lore builder wise, you will want to avoid using actual human racial terms which tells of their genetic history. Because it is an alien race and this kind of thinking can give a guy a painful headache. 

Normally its cool to do 'Furries in space!" but with our community, we have neither attempted to base the universe in this direction nor is it safe to do so with our crowd. So please reconsider giving them a more alien name and description. As a furry, it hurts me to say that, but logically this feels like a better action.

Fox-McCloud commented:
Update:

- Added in 10 more hairstyles
- Added in 8 facial hair styles
- Changed the karma cost to 15, based on some feedback
- removed the speed boost from them and changed it so they cannot be encumbered, instead.

MactoPerFuror commented:
Not the biggest deal in the world. Though the name is still kind of awful.

Fox-McCloud commented:
Posted some more pics showing off the new hairstyles and facial fluffs

fj45 commented:
Scrap anything to do with movement speed entirely, it just messes with balance too much.

Without encumbrance, this race is going to be able to jet around in a hardsuit with a duffel bag.

Finalsong commented:
Can you please stop attempting to pass these off as being canines in general? They're very much Vulpine in appearance, and I see next to no features that would point otherwise.

Spacemanspark commented:
...
On the fence.
I'll go with a :+1: for now.

Allfd commented:
:+1:

Flip-Sandwich commented:
Eh. Not the hugest fan of the sprites.

The first two images just seem... off and wrong, the vertical ears okay on taj that sorta worked? ish? they were small enough to not be that much of an issue but here the hugeness of the ears combined with the fact that they point straight up.

There's a bit of an oddity with the nose, the nose looks like a mouth on the front sprite aaa.

And yes hardsuit duffelbag, gotta go heavy and still go fast because can't go slow.

and finally, vulpkanin vulpine canine vulpinekin what

tigercat2000 commented:
I will just leave this here:
Everyone seems to disagree with all of the balance adjustments of this race

In my opinion
If you removed everything but the language and set the heat/cold resistance to the same as tajaran, and made them cost no karma, avalible to everyone, this race would actually be a pretty cool idea

Spacemanspark commented:
The heads on the first few pictures look like a really squashed Eevee.

Iamgoofball commented:
Hold on

You realize that ignoreslowdown means I move at default run speed regardless of any damage type, right? 

And that lowering the cost is an even bigger buff?

This is still insanely overpowered, and really abusable.

Stop dancing around the issue. Increase the damage taken even more.

Finalsong commented:
@Iamgoofball What he said. Also, ignoreslowdown with this speed is so unbelievably powerful that I can't even

Iamgoofball commented:
PS. I can just use riot armor to just deal with the current measly damage buffs. And guess what, because of the ignoreslowdown, the downside to riot armor(reduced speed) stops being a thing.

New meta: police dogs, literally

Fox-McCloud commented:
- updated the the top pic with one provided by Full of Skittles that better showcases the various hairstyles.
- remove all bonsues
- Marked as DNM until some niche power can be designed for them along with a feasible downside.

Spacemanspark commented:
Eh, I like it at 17x 17, I can live with it.
And I'm pretty sure if you start a forum topic, you'll get some ideas.

Fox-McCloud commented:
Spoke with Neca.

- removed karma cost
- keeping their abilities basically the same as Tajarans.
